sci-mathematics/rstudio/files/rstudio-1.4.1106-soci-cmake-find_library.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/src/cpp/CMakeLists.txt b/src/cpp/CMakeLists.txt
```

Comment:
I'd consider dropping the first two lines but it's not needed either.

Reply:
For better or worse, I use the raw `git diff` as described [here on the wiki](https://wiki.gentoo.org/wiki/Patches#Creating_a_patch). Do you think the wiki should be updated?

Reply:
[devmanual](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto) is authoritative but yes

Reply:
ok, duly noted, I'll rework all the patches to match the criteria laid out in the devmanual.

Reply:
Patches are fixed and should conform to the layout suggested in the devmanual. Closing as resolved.